testing/integration-testing-hooks.ts

```diff
@@ -507,6 +507,21 @@ export function skipIfCommunityServer(server: MongodSetup): void {
   });
 }
 
+/**
+ * Skip tests in the suite if the test server is an enterprise server.
+ *
+ * describe('...', () => {
+ *   e.g. skipIfEnterpriseServer(testServer)
+ * });
+ */
+ export function skipIfEnterpriseServer(server: MongodSetup): void {
+  before(async function() {
+    if (!await server.isCommunityServer()) {
```

Comment:
Definitely a small not really doing anything suggestion, so feel free to ignore. Maybe we could add a helper to be more explicit about this `community` vs `enterprise` check on the [mongod setup class](https://github.com/mongodb-js/mongosh/blob/011ee10a0d505b7ec2e9af63c16f365e2e2c1373/testing/integration-testing-hooks.ts#L210). I'm thinking way down the line there could be something besides community that isn't enterprise and having it more explicitly set there would make that check more transparently updated.
```suggestion
    if (await server.isEnterpriseServer()) {
```

Reply:
We could do that, but I think if/when that happens we’ll need to revisit all our `skipIfEnterpriseServer()`/`skipIfCommunityServer()` calls anyway, so I think it’s okay to just wait rather than over-planning for the future?

Reply:
sgtm!

Reply:
Fwiw, I had to remove this function anyway because this check requires the server to be running, but the `--setParameter` flags in the test prevent it from starting in the first place if it’s an enterprise server.